opentelemetry-etw-metrics/src/exporter/mod.rs

```diff
@@ -0,0 +1,87 @@
+use opentelemetry::metrics::{MetricsError, Result};
+use opentelemetry_proto::tonic::collector::metrics::v1::ExportMetricsServiceRequest;
+use opentelemetry_sdk::metrics::{
+    data::{ResourceMetrics, Temporality},
+    exporter::PushMetricsExporter,
+    reader::{AggregationSelector, DefaultAggregationSelector, TemporalitySelector},
+    Aggregation, InstrumentKind,
+};
+
+use async_trait::async_trait;
+use prost::Message;
+
+use std::fmt::{Debug, Formatter};
+
+use crate::etw;
+
+pub struct MetricsExporter {}
+
+impl MetricsExporter {
+    pub fn new() -> MetricsExporter {
+        etw::register();
+
+        MetricsExporter {}
+    }
+}
+
+impl Default for MetricsExporter {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl TemporalitySelector for MetricsExporter {
+    fn temporality(&self, kind: InstrumentKind) -> Temporality {
+        match kind {
+            InstrumentKind::Counter
+            | InstrumentKind::ObservableCounter
+            | InstrumentKind::ObservableGauge
+            | InstrumentKind::Histogram
+            | InstrumentKind::Gauge => Temporality::Delta,
+            InstrumentKind::UpDownCounter | InstrumentKind::ObservableUpDownCounter => {
+                Temporality::Cumulative
+            }
+        }
+    }
+}
+
+impl AggregationSelector for MetricsExporter {
+    fn aggregation(&self, kind: InstrumentKind) -> Aggregation {
+        DefaultAggregationSelector::new().aggregation(kind)
+    }
+}
+
+impl Debug for MetricsExporter {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.write_str("ETW metrics exporter")
+    }
+}
+
+#[async_trait]
+impl PushMetricsExporter for MetricsExporter {
+    async fn export(&self, metrics: &mut ResourceMetrics) -> Result<()> {
+        let proto_message: ExportMetricsServiceRequest = (&*metrics).into();
+
+        let mut byte_array = Vec::new();
+        proto_message
+            .encode(&mut byte_array)
+            .map_err(|err| MetricsError::Other(err.to_string()))?;
+
+        match etw::write(&byte_array) {
+            0 => println!("Successfully wrote ETW event"),
```

Comment:
nit - Is this `println!` for debug purpose, as this will be logging each successful write? If we intend to keep it, should we use instead tokio tracing [debug](https://docs.rs/tracing/latest/tracing/macro.debug.html) macro?

Reply:
we can remove the println all the places in this PR. and use global error handler for any error case. 
Can't use tracing macros unless we make that as common thing across the repo and solve live-loop etc.